tests/sqlparser_mssql.rs

```diff
@@ -2053,3 +2053,37 @@ fn parse_drop_trigger() {
         }
     );
 }
+
+#[test]
+fn parse_print() {
+    let print_string_literal = "PRINT 'Hello, world!'";
+    let print_stmt = ms().one_statement_parses_to(print_string_literal, "");
+    assert_eq!(
+        print_stmt,
+        Statement::Print(PrintStatement {
+            message: Box::new(Expr::Value(
+                (Value::SingleQuotedString("Hello, world!".to_string())).with_empty_span()
+            )),
+        })
+    );
+
+    let print_national_string = "PRINT N'Hello, ⛄️!'";
+    let print_stmt = ms().one_statement_parses_to(print_national_string, "");
+    assert_eq!(
+        print_stmt,
+        Statement::Print(PrintStatement {
+            message: Box::new(Expr::Value(
+                (Value::NationalStringLiteral("Hello, ⛄️!".to_string())).with_empty_span()
+            )),
+        })
+    );
+
+    let print_variable = "PRINT @my_variable";
+    let print_stmt = ms().one_statement_parses_to(print_variable, "");
+    assert_eq!(
+        print_stmt,
+        Statement::Print(PrintStatement {
+            message: Box::new(Expr::Identifier(Ident::new("@my_variable"))),
+        })
+    );
```

Comment:
oh was there a requirement to use `one_statement_parses_to` instead of `verified_stmt` for these tests?

Also with vierified_stmt we can simplify the tests except, for the hello world example, to skip assertion on the AST

Reply:
I'm unaware of any requirement, it's just that I don't understand the project testing conventions & I'm trying to follow the patterns already present in the file. I'll switch it over to your suggestion

Reply:
I think we might want to inspect the ast in the future if we do an enum of value params instead of an expr, but we can cross that bridge if/when we ever come to it. This suggestion is complete 👍